src/Cache/DefaultCacheFactory.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Eppo\Cache;
+
+use Exception;
+use Psr\SimpleCache\CacheInterface;
+use Sarahman\SimpleCache\FileSystemCache;
+
+class DefaultCacheFactory
+{
+    /**
+     * @throws Exception
+     */
+    public static  function create(): CacheInterface
+    {
+        return new FileSystemCache(__DIR__ . '/../../cache/');
```

Comment:
I wonder if we should use the operating systems temporary directory or something

Reply:
done